test/lnd_services_mock.go

```diff
@@ -127,6 +130,7 @@ type LndMockServices struct {
 
 	Height     int32
 	NodePubkey string
+	Signature  []byte
```

Comment:
One could still debate whether it should be here or on the mock. But not blocking